packages/react-instantsearch/src/connectors/connectMenu.js

```diff
@@ -68,22 +72,21 @@ export default createConnector({
     const {results} = searchResults;
     const {attributeName, showMore, limitMin, limitMax} = props;
     const limit = showMore ? limitMax : limitMin;
-
     const canRefine =
       Boolean(results) &&
       Boolean(results.getFacetByName(attributeName));
 
     const isFromSearch = Boolean(searchForFacetValuesResults
       && searchForFacetValuesResults[attributeName]
       && searchForFacetValuesResults.query !== '');
-    const searchForFacetValues = props.searchForFacetValues ? this.searchForFacetValues : undefined;
+    const withSearchBox = props.withSearchBox || props.searchForFacetValues;
 
     if (!canRefine) {
       return {
         items: [],
         currentRefinement: getCurrentRefinement(props, searchState),
         isFromSearch,
-        searchForFacetValues,
+        withSearchBox,
```

Comment:
This will break for people using props.searchForFacetValues in their connector usage no? We need to pass both properties sadly.